gTea/app/src/main/java/com/nechaieva/gtea/NavigationFragment.java

```diff
@@ -30,5 +34,19 @@ public void onClick(View view) {
                         .navigate(R.id.action_NavFragment_to_SecondFragment);
             }
         });
+
+        RecyclerView recyclerView = (RecyclerView) view.findViewById(R.id.menu_items);
+
+        // use this setting to improve performance if you know that changes
+        // in content do not change the layout size of the RecyclerView
+        recyclerView.setHasFixedSize(true);
```

Comment:
Consider removing the optimisation for simplicity.

Reply:
Yeah, it doesn't seems to change much; removed.